src/components/ItaliaTheme/View/Commons/RelatedItems.jsx

```diff
@@ -82,7 +82,9 @@ const RelatedItems = ({
                   <>
                     <Row>
                       <Col className="text-center">
-                        <h3>{intl.formatMessage(messages.related_items)}</h3>
+                        <p className="h3">
+                          {intl.formatMessage(messages.related_items)}
+                        </p>
```

Comment:
Qui non sono sicuro sia corretto, è una sezione dedicata ai contenuti correlati e in quanto tale si merita un titolo direi. Se il problema è di ordinamento, hai provato per caso a fare `<h2 className="h3">`? Se ricordo bene i titoli di sezione sono h2 quindi se la mia intuizione è corretta questo potrebbe risolvere.
In caso parliamone a voce.